.bazelrc

```diff
@@ -94,7 +94,7 @@ build:asan --linkopt -fsanitize=address
 test:asan --jobs=1
 test:asan --test_env=ASAN_OPTIONS="detect_leaks=0"
 # This LD_PRELOAD is set for Travis. You will need to change it for local debugging.
-test:asan --test_env=LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libasan.so.2 /usr/lib/gcc/x86_64-linux-gnu/7/libasan.so"
+test:asan --test_env=LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libasan.so.5 /usr/lib/gcc/x86_64-linux-gnu/9/libasan.so"
```

Comment:
who is the relevant person to review this? cc @ericl ?

Reply:
FWIW these are only the asan tests and they only seem to be run manually. The same change has been done for buildkite:

```
test:asan-buildkite --test_env=LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libasan.so.5"
```